app/src/main/java/ru/practicum/android/diploma/presentation/viewmodels/VacancyViewModel.kt

```diff
@@ -0,0 +1,75 @@
+package ru.practicum.android.diploma.presentation.viewmodels
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import ru.practicum.android.diploma.domain.DetailsInteractor
+import ru.practicum.android.diploma.domain.db.FavouriteVacanciesInteractor
+import ru.practicum.android.diploma.domain.models.VacancyDetails
+import ru.practicum.android.diploma.ui.state.FavouriteState
+import ru.practicum.android.diploma.ui.state.VacancyScreenState
+import ru.practicum.android.diploma.util.ResponseData
+
+class VacancyViewModel(
```

Comment:
```suggestion
class VacancyDetailsViewModel(
```
кажись так будет информативнее

Reply:
а может это и лишнее даже, хз